vagrant/provisioning/setup-master.sh

```diff
@@ -40,9 +40,26 @@ HA="false"
 # FIXME(mestery): Remove once Vagrant boxes allow apt-get to work again
 sudo rm -rf /var/lib/apt/lists/*
 
-# Add external repos to install docker and OVS from packages.
+# Install CNI
+pushd ~/
+wget -nv https://github.com/containernetworking/cni/releases/download/v0.5.2/cni-amd64-v0.5.2.tgz
+popd
+sudo mkdir -p /opt/cni/bin
+pushd /opt/cni/bin
+sudo tar xvzf ~/cni-amd64-v0.5.2.tgz
+popd
+sudo mkdir -p /etc/cni/net.d
+# Create a 99loopback.conf to have atleast one CNI config.
+echo '{
```

Comment:
I don't think we need this, but doesn't hurt I guess.

Reply:
In my experience, kubelet is "NotReady" without a CNI plugin. So it may not be able to deploy daemonsets which in our case will deploy the plugin.